tests/nosetests/pyanaconda_tests/product_test.py

```diff
@@ -221,6 +221,11 @@ def fedora_products_test(self):
             ["fedora.conf", "fedora-iot.conf"],
             WORKSTATION_PARTITIONING
         )
+        self._check_default_product(
+            "Fedora", "ELN",
+            ["fedora.conf", "fedora-eln.conf"],
+            SERVER_PARTITIONING
```

Comment:
When you have `SERVER_PARTITIONING` then maybe the variant should be inherited from server?

If yes, then please add.
```
[Base Product]
...
variant_name = Server
```

and remove unnecessary parts (most of the configuration).

Reply:
I don't really know, but shouldn't the configuration of Fedora ELN be closer to RHEL than Fedora Server?

Reply:
> I don't really know, but shouldn't the configuration of Fedora ELN be closer to RHEL than Fedora Server?

Actually, that's a really good point. I'll see about doing that.

> When you have `SERVER_PARTITIONING` then maybe the variant should be inherited from server?
> 
> If yes, then please add.
> 
> ```
> [Base Product]
> ...
> variant_name = Server
> ```

Could you point me at documentation for that? I must have missed it.

Reply:
Basically, we support simple inheritance of the product configuration files:
https://anaconda-installer.readthedocs.io/en/latest/configuration-files.html#id1

I wonder whether Fedora ELN shouldn't be a base product of all RHEL-based products.

Reply:
It should not. It's not something stable and I would like to avoid having issues with inheritance because ELN have to be hacked to work.